edx_exams/apps/api/v1/views.py

```diff
@@ -271,3 +274,116 @@ class ProctoringProvidersView(ListAPIView):
     model = ProctoringProvider
     serializer_class = ProctoringProviderSerializer
     queryset = ProctoringProvider.objects.all()
+
+
+class ExamAccessTicketsView(APIView):
+    """
+    View to create signed exam access tickets for a specific user and exam.
+
+    Given an exam_id and user (in request), this view will either grant access
+    as an exam access ticket or not grant access. Access is granted if there is an
+    existing exam attempt or if the exam is past its due date.
+
+    HTTP GET
+    Provides an Exam Access Ticket as a cookie if access is granted.
+    **GET data Parameters**
+        * exam_id: This is the id of the exam that the user is requesting access to
+    **Exceptions**
+        * HTTP_403_FORBIDDEN
+    """
+
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (StaffUserOrReadOnlyPermissions,)
+
+    @classmethod
+    def get_current_exam_attempt(cls, user, exam):
+        """
+        Given a user and exam, get the user's latest exam attempt, if exists.
+        """
+        try:
+            exam_attempt = ExamAttempt.objects.filter(user=user, exam=exam).latest('created')
+        except ObjectDoesNotExist:
+            exam_attempt = None
+        return exam_attempt
+
+    @classmethod
+    def set_expiration_window(cls, exam_attempt, default_exp_seconds):
```

Comment:
Looks like this is more of a `get_expiration_window` or a `calculate_expiration_window` since it just returns a value